app/views/partials/_nav.html.erb

```diff
@@ -59,37 +59,26 @@
   <div class="nav">
     <div class="container-lg">
       <a href="/" class="logo"></a>
-
-      <form class="discussion-search">
```

Comment:
They still want to include search in the discussions page! just not have it in the nav bar - so I think the best spot is in the light blue section (make the light blue section a bit thicker and add in the search bar above where the tags are)

![screen shot 2017-04-05 at 2 24 25 pm](https://cloud.githubusercontent.com/assets/10657390/24727658/96e22152-1a0b-11e7-84ab-54f86b88a382.png)